Schedule_app/urls.py

```diff
@@ -15,5 +15,6 @@
     path('choose_seats/', views.seats, name='seats'),
     path('confirmation/', views.confirmation, name='confirmation'),
     path('confirmbooking/<uidb64>/<token>/', ConfirmBooking.as_view(), name='confirmbooking'),
+    path('download_bookings', views.download_csv, name='download_bookings'),
```

Comment:
Try to use `-` instead of `_` in urls. You can read more about it online if you want. (https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)